tests/ui-cargo/multiple_config_files/warn/src/main.stderr

```diff
@@ -0,0 +1,3 @@
+// ignore-windows
```

Comment:
The CI complains about this comment. Doesn't this have to be added to the `.rs` test file? :upside_down_face:

Reply:
Ah good call :D